homeassistant/components/vicare/climate.py

```diff
@@ -217,7 +217,9 @@ def update(self):
                     "heating_curve_shift"
                 ] = self._circuit.getHeatingCurveShift()
 
+            # Provide vicare_modes and vicare_programs for debugging purposes
```

Comment:
If these are just for debugging, why not log them at debug level instead?

Reply:
Currently it's only for debugging but just as with vicare_modes we will need this data to check which presets the device actually supports.
A dedicated PR will follow soon but right now I'm still collecting data from users to understand how to deal the mapping from hvac presets to vicare programs.

I could still remove it from the attributes if you have hard feelings about that.

Reply:
I think they should be removed, the state machine is not for storing debug data.